src/libs/shim-services/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "shim-services"
```

Comment:
I think it would be helpful somewhere to state that we're talking about the runtime-rs (runtime) shim to avoid confusion with any other shims ;) It's the same sort of issue as when someone mentions "proxy", "handler", "interface", "protocol", I always think, "which one?" ;)

Reply:
I think the interface would be more generic and appropriate. Since this works as a interface facing outside binaries.